apps/fluent-tester/src/FluentTester/TestComponents/ButtonExperimental/ButtonShapeTestSection.tsx

```diff
@@ -0,0 +1,17 @@
+import { Button } from '@fluentui-react-native/experimental-button';
+import * as React from 'react';
+import { View } from 'react-native';
+import { commonTestStyles, stackStyle } from '../Common/styles';
+
+export const ButtonShapeTest: React.FunctionComponent = () => {
+  return (
+    <View style={[stackStyle, commonTestStyles.view]}>
+      <Button shape="rounded" content="Default Rounded Button" />
+      <Button shape="square" content="Square Button" />
+      <Button shape="circular" content="Circular Button" />
+      <Button appearance="primary" shape="rounded" content="Default Rounded Button" />
```

Comment:
nit: Maybe instead of "Default" we should have "Primary" for the content?

Reply:
Yes, will fix